Before: 42d2ac257efa9e319132158f2db5ea29e9971aab5fd3ec1b0cc365b679b58d99
After: b9cd900bf090b58c8c6d308ff87f21546a770c01df925d6636fae811ba5d3b92
Patch: 17705be08f42c8a26531cc4a7958d9238e44546b1e18db1f56355ddb90a4bdcb
--- bin/%plugin_name%.tt	2012-08-15 14:36:44.289437698 +1000
+++ bin/%plugin_name%.tt	2012-08-15 14:36:48.832793884 +1000
@@ -14,8 +14,8 @@
 require 'bundler/setup' if ENV['BUNDLE_GEMFILE']
 
 # Start CLI
-require 'corvid/cli/plugin'
-Corvid::CLI::Plugin.start project_root:        project_root,
-                          plugin_require_path: '<%= require_path %>',
-                          plugin_class_name:   '<%= class_name %>'
+require 'corvid/plugin_cli'
+Corvid::PluginCli.start project_root:        project_root,
+                        plugin_require_path: '<%= require_path %>',
+                        plugin_class_name:   '<%= class_name %>'
 
